src/views/dashboard/User.tsx

```diff
@@ -2,54 +2,47 @@ import React, { useState, useEffect } from "react";
 import { Query, QueryResult } from "react-apollo";
 import { GET_USER } from "../../graphql-requests/queries";
 import "../home/Home.css";
-import TextField from "@material-ui/core/TextField";
 //import { makeStyles } from "@material-ui/core/styles";
 
+//let user_id = localStorage.getItem("user_id");
+
 const User: React.FC = (props: any) => {
    const [user_id, setUser] = useState("");
+
    useEffect(() => {
       let user_id = localStorage.getItem("user_id");
       if (user_id) {
          setUser(user_id);
       }
-   }, []);
+   }, [user_id]);
```

Comment:
I believe this will cause an infinite loop if `user_id` is associated with the component-level state variable